src/model/debug/debugobjecttreemodel.cpp

```diff
@@ -0,0 +1,190 @@
+/* SPDX-License-Identifier: GPL-3.0-or-later
+ * Copyright © 2024 The TokTok team.
+ */
+
+#include "debugobjecttreemodel.h"
+
+// explicit DebugObjectTreeModel(QObject* parent = nullptr);
```

Comment:
What's up with these comments?

Reply:
That was to remind myself of the functions. Deleted now.